### PR TITLE
admintool31zip_1f965184

### PR DESCRIPTION
**Pull request by unreal-archive-submitter**
Add content: 
 - [UT2004 MUTATOR] Akimbo
 - [UT99 MAP] MH-Assasination
 - [UT2004 MUTATOR] Replaces most of the weapons and their ammo with brand new toys! (and ammo)

---
Submission log: https://unrealarchive.org/submit/#1f965184

### CODE REVIEW

File: content/Unreal Tournament 2004/Mutators/R/4/b/d52473/replaces-most-of-the-weapons-and-their-ammo-with-brand-new-toys-and-ammo_[4bd52473].yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+--- !<MUTATOR>
+contentType: "MUTATOR"
+firstIndex: "2023-03-15 23:53"
+game: "Unreal Tournament 2004"
+name: "Replaces most of the weapons and their ammo with brand new toys! (and ammo)"
+author: "Unknown"
+description: ""
```
Comment (NeonKnightOA) [inline content/Unreal Tournament 2004/Mutators/R/4/b/d52473/replaces-most-of-the-weapons-and-their-ammo-with-brand-new-toys-and-ammo_[4bd52473].yml:7]: description: "Replaces most of the weapons and their ammo with brand new toys! (and ammo)"

File: content/Unreal Tournament 2004/Mutators/R/4/b/d52473/replaces-most-of-the-weapons-and-their-ammo-with-brand-new-toys-and-ammo_[4bd52473].yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- url: "https://f002.backblazeb2.com/file/unreal-archive-files/Unreal%20Tournament%202004/Mutators/R/4/b/d52473/CG2k3.zip"
+  main: true
+  repack: false
+  state: "OK"
+links: {}
+deleted: false
+mutators:
+- name: "Replaces most of the weapons and their ammo with brand new toys! (and ammo)"
+  description: ""
+weapons: []
```
Comment (NeonKnightOA) [inline content/Unreal Tournament 2004/Mutators/R/4/b/d52473/replaces-most-of-the-weapons-and-their-ammo-with-brand-new-toys-and-ammo_[4bd52473].yml:29]: The weapons are all found in the Readme.

File: content/Unreal Tournament 2004/Mutators/R/4/b/d52473/replaces-most-of-the-weapons-and-their-ammo-with-brand-new-toys-and-ammo_[4bd52473].yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+--- !<MUTATOR>
+contentType: "MUTATOR"
+firstIndex: "2023-03-15 23:53"
+game: "Unreal Tournament 2004"
+name: "Replaces most of the weapons and their ammo with brand new toys! (and ammo)"
+author: "Unknown"
+description: ""
+releaseDate: "2003-04"
```
Comment (NeonKnightOA) [inline content/Unreal Tournament 2004/Mutators/R/4/b/d52473/replaces-most-of-the-weapons-and-their-ammo-with-brand-new-toys-and-ammo_[4bd52473].yml:8]: releaseDate: "2003-04-20"

File: content/Unreal Tournament 2004/Mutators/A/2/4/5907c9/akimbo_[245907c9].yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+--- !<MUTATOR>
+contentType: "MUTATOR"
+firstIndex: "2023-03-15 23:53"
+game: "Unreal Tournament 2004"
```
Comment (NeonKnightOA) [inline content/Unreal Tournament 2004/Mutators/A/2/4/5907c9/akimbo_[245907c9].yml:4]: game: "Unreal Tournament 2003"

File: content/Unreal Tournament 2004/Mutators/A/2/4/5907c9/akimbo_[245907c9].yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+name: "Akimbo"
+author: "Unknown"
```
Comment (NeonKnightOA) [inline content/Unreal Tournament 2004/Mutators/A/2/4/5907c9/akimbo_[245907c9].yml:6]: author: "jasonyu"

**Comment by shrimpza** [comment]
merged via another commit

**Pull request closed without merging**